.azure-devops/create-release.yml

```diff
@@ -31,9 +31,9 @@ parameters:
 - name: 'testHub'
   type: boolean
   default: true
-- name: azureSubscription
-  type: string
-  default: 'AzIoTCLIService'
+
+variables:
+  - group: aziotcli_test_primary
```

Comment:
Possible to parameterize the variable group value as well?

Reply:
Certainly possible to parametrize this as a string - might wanna look at how this affects the YAML validation at runtime, but I don't think it would be an issue. Makes more sense to parametrize here rather than the nightly pipeline, since we're always running release ad-hoc. I'll try it out.